vita3k/CMakeLists.txt

```diff
@@ -99,6 +99,9 @@ add_subdirectory(gdbstub)
 
 add_executable(vita3k MACOSX_BUNDLE main.cpp interface.cpp interface.h performance.cpp)
 
+# Set Visual Studio startup project
+set_property(DIRECTORY .. PROPERTY VS_STARTUP_PROJECT "vita3k")
```

Comment:
i no understund this ? i have already fixed this things before

Reply:
Setting the startup project is only handled for windows systems in https://github.com/Vita3K/Vita3K/blob/master/vita3k/CMakeLists.txt#L167 tho, this change is outside that if and it sets the startup project globally, altough i don't know if setting the startup project twice (in case for windows systems) is ok, VSCode Cmake does this automatically :man_shrugging:

Reply:
> Setting the startup project is only handled for windows systems in https://github.com/Vita3K/Vita3K/blob/master/vita3k/CMakeLists.txt#L167 tho, this change is outside that if and it sets the startup project globally, altough i don't know if setting the startup project twice (in case for windows systems) is ok, VSCode Cmake does this automatically 🤷‍♂️

is what i have fixed yes, this line, so no need add another like is already fine here

Reply:
Just dropped the commit. :+1: